install.sh

```diff
@@ -1,48 +1,19 @@
-#!/bin/bash
-set -e
-echo "=====================================Welcome to cloud k8s cluster installer====================================="
-echo "Usage: ./install.sh NUM_OF_NODES=3 CLOUD_PROVIDER=gcp GCP_PROJECT_NAME=project-name-value"
-SUPPORTED_CLOUD_PROVIDERS=("gcp")
-NUM_OF_NODES=3
-DEFAULT_CLOUD_PROVIDER="gcp"
-for ARGUMENT in "$@"
-do
-
-    KEY=$(echo $ARGUMENT | cut -f1 -d=)
-    VALUE=$(echo $ARGUMENT | cut -f2 -d=)   
-
-    case "$KEY" in
-            NUM_OF_NODES) NUM_OF_NODES=${VALUE} ;;
-            CLOUD_PROVIDER) CLOUD_PROVIDER=${VALUE} ;; 
-            GCP_PROJECT_NAME) GCP_PROJECT_NAME=${VALUE} ;;    
-            *)   
-    esac    
-
+#!/bin/bash -e
 
-done
-
-if [ $NUM_OF_NODES -lt 1 ]; then
-echo "NUM_OF_NODES cannot be less than 1"
-exit 1
-fi
+echo "=====================================Welcome to cloud k8s cluster installer====================================="
+echo "Usage: NUM_OF_NODES=3 CLOUD_PROVIDER=gcp GCP_PROJECT_NAME=project-name-value ./install.sh"
```

Comment:
Request you to change the same in Readme.md.

Reply:
done